services/postgres/serverless.yml

```diff
@@ -181,7 +184,11 @@ resources:
         UserData:
           Fn::Base64: !Sub |
             #!/bin/bash
-            apt update && apt install unzip postgresql postgresql-contrib -y
+            # get apt data for postgres-14
+            sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
```

Comment:
Don't really need a sudo here as everything runs as root. 

One thing though is that the user data scripts only get run once and it's at time of creation, so we should make sure we cycle the machines so this gets applied on dev/val/prod VMs.

Reply:
I've already run this on the existing jump boxes, should we be cycling these automatically?